src/current/v23.1/cloud-storage-authentication.md

```diff
@@ -707,82 +708,23 @@ You can authenticate to Azure with explicit credentials in the following ways:
 Implicit authentication to Azure is only available for CockroachDB {{ site.data.products.core }} clusters.
 {{site.data.alerts.end}}
 
-{% include_cached new-in.html version="v23.1.9" %} When the `AUTH` parameter is set to `implicit`, CockroachDB will load credentials from one of the following:
```

Comment:
Do you mean to revert the changes of PR [18119](https://github.com/cockroachdb/docs/pull/18119)?